pkg/web/src/app/queries/helpers.ts

```diff
@@ -170,9 +170,11 @@ export const truncateK8sString = (prefix: string, suffix = '', generateName = fa
   return `${prefix.slice(0, targetPrefixLength)}${suffix}`;
 };
 
+const ALL_NAMESPECES = 'all-namespaces';
```

Comment:
you can import it from the consts file ?

Reply:
This is cross package dependency (legacy code vs code in `src`).

Reply:
NICE !